scripts/trino-acl

```diff
@@ -49,8 +50,20 @@ def create_in_acl(acl, type, name, user_type, resource_name, allow, privileges):
     return acl
 
 
-def update_acl(acl, type, name, user_type, resource_name, allow):
-    print('This still needs to be implemented')
+def update_acl(acl, type, search_field, search_field_value, update_field, update_field_value, update_field_replacement):
+    try:
```

Comment:
there is a lot of tricky string manipulation going on in here.  some doc on what is actually being performed would be helpful IMO

Reply:
(in particular, the expected format of the variables we're working on would be helpful in maintaining this down the line)

Reply:
I've added the pydoc in the same PR, let me know if it's more clear, if not ill add a doc discussing the implementation. Best way to explain it is by checking the config and giving some examples updating some fields